<launch>

    <!-- <arg name="demo_name" default="picknplace"/> -->
    <arg name="working_dir" value="$(find lfd_smoothing)/data" />
    <arg name="demo_name" default="picknplace0"/> 
    <arg name="traj_arg" default="smooth"/> 
    <arg name="traj_duration" default="0"/> 

    <include file="$(find lfd_interface)/launch/lfd_storage.launch"/>

    <node name="execute_trajectory" pkg="lfd_smoothing" type="execute_trajectory.py" output="screen">
        <param name="demo_name" value="$(arg demo_name)"/>
        <param name="traj_arg" value="$(arg traj_arg)"/>
        <param name="traj_duration" value="$(arg traj_duration)"/>
        <param name="working_dir" value="$(arg working_dir)"/>
    </node>

</launch>
